CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
 # Changelog
+## [V2.10.0](https://bolt-public.s3-us-west-1.amazonaws.com/magento-integration-release/bolt-magento1-v2.10.0.zip) 2020-11-05
+ - added: custom server URLs for staging and dev are now supported.
```

Comment:
```suggestion
 - Added support for custom URLs for staging and dev
```

Reply:
@adentong This is a new approach we already use in M2 for a few months
https://github.com/BoltApp/bolt-magento2/blob/master/CHANGELOG.md
CC: @lbliii

Reply:
@vitaliyreznikov updated